certs/sp/demo_sp.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Yes, you are right. Sorry to mix up the words and create even more confusion in an already confusing world!